src/renderers/art/ReactARTFiber.js

```diff
@@ -485,6 +485,10 @@ const ARTRenderer = ReactFiberReconciler({
     // Noop
   },
 
+  getChildHostContext() {
+    return null;
+  },
+
```

Comment:
nit: I think it would be nice for us to alpha-sort `HostConfig` props to make it easier to scan for differences between fiber renderers. (At least it helps when things are changing fairly often due to fiber being in-flux.)